ote/resources/public/language/fi.edn

```diff
@@ -149,7 +149,7 @@
 
   :ote.db.transport-service/information-service-accessibility
   {:audio-for-hearing-impaired "Ääniopastus kuulovammaisille"
-   :audio-information "Äänelliset informaatiot"
+   :audio-information "Kuulutukset"
```

Comment:
This is not necessarily "kuulutukset". What about billboards that provide audio guidance when pressing a button etc. But, this is good enough for publishing.